### PR TITLE
Regenerate error reporting

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

File: generated/python/gapic-google-cloud-error-reporting-v1beta1/google/cloud/gapic/errorreporting/v1beta1/error_stats_service_client.py [hunk 1]
```diff
@@ -33,14 +33,14 @@
 import google.gax
 
 from google.cloud.gapic.errorreporting.v1beta1 import enums
-from google.devtools.clouderrorreporting.v1beta1 import error_stats_service_pb2
+from google.cloud.grpc.devtools.clouderrorreporting.v1beta1 import error_stats_service_pb2
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-error-reporting-v1beta1/google/cloud/gapic/errorreporting/v1beta1/error_stats_service_client.py]: The gRPC path should be `google.cloud.grpc.errorreporting.v1beta1`, which is analogous to the GAPIC path of `google.cloud.gapic.errorreporting.v1beta1`.

**Comment by bjwatson** [comment]
Approving this for now to unblock @waprin, but we should fix the gRPC path in a subsequent release.

**Comment by geigerj** [comment]
Tracking gRPC package renaming as https://github.com/googleapis/artman/issues/133

**Comment by waprin** [comment]
Thanks, ping me when released.

**Pull request merged**